packages/action-menu/src/ActionMenu.ts

```diff
@@ -87,6 +87,7 @@ export class ActionMenu extends ObserveSlotPresence(
     protected override render(): TemplateResult {
         return html`
             <sp-action-button
+                aria-describedby=${ifDefined(this.description)}
```

Comment:
This should be statically applied, `ifDefined()` shouldn't be needed.

Reply:
Done!